### ISSUE TITLE
intersect fails using x-range wildcard

### ISSUE THREAD

**Issue by cdaringe** [npm/node-semver#266]
# problem

```js
var semver = require("semver")
semver.intersects('^1.0.0', '*')
// at new SemVer in semver/semver.js — line 296
//   throw new TypeError('Invalid Version: ' + version);
//   TypeError: Invalid Version: [object Object]
```

expected `true`

**Comment by Asthetic** [comment]
Created a PR that appears to fix this.
https://github.com/npm/node-semver/pull/271

**Comment by isaacs** [comment]
Fixed in 6.1.0
